packages/cisco_meraki/kibana/dashboard/cisco_meraki-4832a430-af22-11ec-a899-6f7e676e0fb4.json

```
@@ -1102,35 +1324,63 @@
                 },
                 "panelIndex": "58d65007-15fc-492f-a8db-f509b7d28aad",
                 "title": "Top URL access",
-                "type": "lens",
-                "version": "7.17.0"
+                "type": "lens"
             },
             {
                 "embeddableConfig": {
+                    "attributes": {
+                        "columns": [],
+                        "grid": {},
+                        "hideChart": false,
+                        "isTextBasedQuery": false,
+                        "kibanaSavedObjectMeta": {
+                            "searchSourceJSON": {
+                                "filter": [],
+                                "indexRefName": "kibanaSavedObjectMeta.searchSourceJSON.index",
+                                "query": {
+                                    "language": "kuery",
+                                    "query": "data_stream.dataset : \"cisco_meraki.log\""
+                                }
+                            }
+                        },
+                        "references": [
+                            {
+                                "id": "logs-*",
+                                "name": "kibanaSavedObjectMeta.searchSourceJSON.index",
+                                "type": "index-pattern"
+                            }
+                        ],
+                        "sort": [
+                            [
+                                "@timestamp",
+                                "desc"
+                            ]
+                        ],
+                        "timeRestore": false,
+                        "usesAdHocDataView": false
+                    },
                     "enhancements": {}
                 },
                 "gridData": {
-                    "h": 20,
-                    "i": "a7fc4a8a-954f-4fc0-acfc-2d358c89b2c6",
+                    "h": 19,
+                    "i": "6d997d34-f35a-492c-adb0-2079ad9006e1",
                     "w": 48,
                     "x": 0,
                     "y": 39
                 },
-                "panelIndex": "a7fc4a8a-954f-4fc0-acfc-2d358c89b2c6",
-                "title": "Log stream",
-                "type": "LOG_STREAM_EMBEDDABLE",
-                "version": "7.17.0"
+                "panelIndex": "6d997d34-f35a-492c-adb0-2079ad9006e1",
+                "title": "Log Stream",
```

```suggestion
                "title": "Logs",
```

To make it consistent with `checkpoint` change.